<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>673</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>673</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculadora do Custo Unitário Básico de Construção (CUB)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/background.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rb_residencial_baixo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>146</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Residencial - Padrão Baixo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_residencial_alto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>192</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Residencial - Padrão Alto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_cal_alto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>238</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Comercial Andares Livres - Padrão Alto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_cal_normal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>215</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Comercial Andares Livres - Padrão Normal</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tipo_projeto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Selecione o Tipo de Projeto: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_projeto_selecionado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_csl_normal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>261</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Comercial Salas e Lojas - Padrão Normal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_csl_alto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>284</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Comercial Salas e Lojas - Padrão Alto</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_galpao_industrial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>307</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Galpão Industrial</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_residencia_popular">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>147</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Residência Popular</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>CALCULADORA DO
CUSTO UNITÁRIO BÁSICO
DE CONSTRUÇÃO (CUB)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_valor_area_m2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>479</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_area_projeto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Digite a área do projeto (m²):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calcular">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 8pt &quot;Monospac821 BT&quot;;
}

QPushButton:hover{
	border: 1px solid rgb(0, 134, 0);
	color: rgb(0, 134, 0);
	font: 8pt &quot;Monospac821 BT&quot;;
}</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fig_m2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/metro.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fig_calculadora">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/calculadora.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fig_logo">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/logomarca.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fig_projeto">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/projeto.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tipo_projeto_selecionado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Tipo  de Projeto Selecionado:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fig_cbic">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/cbic.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cub_m2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Custo Unitário
Básico de Construção:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>610</y>
      <width>551</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_ifpe">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/ifpe.png);border:none!important;
    background-color: white!important;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_custo_final">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospac821 BT</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Monospac821 BT&quot;;border: 1px solid black;
color: rgb(0, 134, 0);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="rb_residencial_normal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Monospac821 BT&quot;;</string>
    </property>
    <property name="text">
     <string>Residencial - Padrão Normal</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fig_custo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/custo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/custo.png</pixmap>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>rb_residencial_baixo</zorder>
   <zorder>rb_residencial_alto</zorder>
   <zorder>rb_cal_alto</zorder>
   <zorder>rb_cal_normal</zorder>
   <zorder>lbl_tipo_projeto</zorder>
   <zorder>lbl_projeto_selecionado</zorder>
   <zorder>rb_csl_normal</zorder>
   <zorder>rb_csl_alto</zorder>
   <zorder>rb_galpao_industrial</zorder>
   <zorder>rb_residencia_popular</zorder>
   <zorder>lbl_titulo</zorder>
   <zorder>txt_valor_area_m2</zorder>
   <zorder>lbl_area_projeto</zorder>
   <zorder>btn_calcular</zorder>
   <zorder>lbl_fig_m2</zorder>
   <zorder>lbl_fig_calculadora</zorder>
   <zorder>lbl_fig_logo</zorder>
   <zorder>lbl_fig_projeto</zorder>
   <zorder>lbl_tipo_projeto_selecionado</zorder>
   <zorder>lbl_fig_cbic</zorder>
   <zorder>lbl_cub_m2</zorder>
   <zorder>rb_residencial_normal</zorder>
   <zorder>lbl_fig_custo</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospac821 BT</family>
    </font>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="font">
     <font>
      <family>Monospac821 BT</family>
     </font>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionlimpar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="limpar">
   <property name="text">
    <string>Limpar</string>
   </property>
  </action>
  <action name="act_limpar">
   <property name="text">
    <string>Limpar</string>
   </property>
  </action>
  <action name="act_sair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionVers_o">
   <property name="text">
    <string>Versão</string>
   </property>
  </action>
  <action name="actionLimpar">
   <property name="text">
    <string>Limpar</string>
   </property>
  </action>
  <action name="act_limpar_2">
   <property name="text">
    <string>Limpar</string>
   </property>
  </action>
  <action name="actionlimpar">
   <property name="text">
    <string>Limpar</string>
   </property>
   <property name="font">
    <font>
     <family>Monospac821 BT</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="font">
    <font>
     <family>Monospac821 BT</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionversao">
   <property name="text">
    <string>Versão</string>
   </property>
  </action>
  <action name="actionversao_2">
   <property name="text">
    <string>versao</string>
   </property>
  </action>
  <action name="actionversao_3">
   <property name="text">
    <string>versao</string>
   </property>
  </action>
  <action name="action_versao">
   <property name="text">
    <string>Versão</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
